scripts/globals/effects/last_resort.lua

```diff
@@ -11,9 +11,12 @@ require("scripts/globals/status");
 -----------------------------------
 
 function onEffectGain(target,effect)
-    target:addMod(dsp.mod.ATTP,15 + target:getMerit(dsp.merit.LAST_RESORT_EFFECT));
-    target:addMod(dsp.mod.DEFP,-15 - target:getMerit(dsp.merit.LAST_RESORT_EFFECT));
-    target:addMod(dsp.mod.HASTE_ABILITY, 156+effect:getPower())
+    target:addMod(dsp.mod.ATTP, 15 + target:getMerit(dsp.merit.LAST_RESORT_EFFECT));
+    
+    effect:setPower(-15 - target:getMerit(dsp.merit.LAST_RESORT_EFFECT) + target:getMod(dsp.mod.LAST_RESORT_DEF_PENALTY));
```

Comment:
gonna hafta do dis differently.
```
The Abyss Sollerets need to remain equipped for this effect to remain active. 
```
also we'd normally just call the effect with the power instead of having its effect override its own power like that. or at least set power to eqqual previous power + bonus.

Reply:
can check it's still equipped in onEffectTick, if not then remove the effect

Reply:
or a latent? Wish I had finished the scriptable latents thing actually.

Reply:
I'm just going to get rid of the mod being added here and add it to where Stalwart Soul calculation happens.

Reply:
No wait. that makes no sense. Sorry, still waking up. Let me finish my coffee first. XD

Reply:
What about onItemCheck?

Reply:
we've been trying to get rid of doing this sorta thing in the item scripts

Reply:
I'd try the ontick first, see how it goes. the prob I am considering is unlikely to happen unless we change it later. testing > all

Reply:
Definitely agree. Not sure how to keep it from feeling delayed, though.

Reply:
I might have to move Last Resort damage taken to core.

Reply:
LATENT_JOB_ABILITY_ACTIVE and then we have one for future situations to.

Reply:
Better. Working on it now.

Reply:
LATENT_STATUS_EFFECT_ACTIVE already exists. XD ez lyfe